src/view.js

```diff
@@ -57,11 +65,15 @@ export default function view (Comp) {
 
         // create a reactive render for the component
         this.render = observe(this.render, {
-          scheduler: () => this.setState({}),
+          scheduler: () => queue.add(this.triggerRender),
           lazy: true
         })
       }
 
+      triggerRender = () => {
```

Comment:
This is HOC which extends user-defined components. It would mess up cases where the user has an own method called `triggerRender`. Use a Symbol instead of a string as the method key to make sure we are not overwriting anything defined by the user.